resqpy/rq_import/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""Functions for RESQML objects related to importing"""
+
+all = [
+    'import_nexus.py', 'GridFromCp.py', 'import_vdb_all_grids.py', 'grid_from_cp.py', 'import_vdb_ensemble.py',
+    'add_ab_properties.py', 'add_surfaces.py'
+]
+
+from .GridFromCp import GridFromCp
```

Comment:
Connor convinced me that we should encourage calling code to import from the package level only by naming all the submodules with a leading underscore, so _GridFromCp.py etc.